templates/element/variables_panel.php

```diff
@@ -24,6 +24,14 @@
     printf('<p class="warning">%s</p>', $error);
 endif;
 
+if (isset($varsMaxDepth)) {
+    $msg = sprintf(__d('debug_kit', '%s levels of nested arrays shown.'), $varsMaxDepth);
+    $msg .= __d('debug_kit', 'You can overwrite this via the config key');
+    $msg .= ' <strong>DebugKit.variablesPanelMaxDepth</strong>';
+    $msg .= __d('debug_kit', ' | Be aware that increasing the level depth can lead to an out of memory error.');
```

Comment:
```suggestion
    $msg .= __d('debug_kit', ' | Be aware that increasing the depth value can lead to an out of memory error.');
```

Reply:
This can be simpler: Increasing the depth value can lead to an out of memory error.

Reply:
I now used the version from @othercorey